src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -23,6 +23,8 @@ The .NET Foundation licenses this file to you under the MIT license.
     <IlcBuildTasksPath Condition="'$(IlcBuildTasksPath)' == ''">$(MSBuildThisFileDirectory)..\tools\netstandard\ILCompiler.Build.Tasks.dll</IlcBuildTasksPath>
     <TargetOS Condition="$(RuntimeIdentifier.StartsWith('win'))">windows</TargetOS>
     <TargetOS Condition="$(RuntimeIdentifier.StartsWith('osx'))">osx</TargetOS>
+    <TargetOS Condition="$(RuntimeIdentifier.StartsWith('ios'))">ios</TargetOS>
+    <TargetOS Condition="$(RuntimeIdentifier.StartsWith('iossimulator'))">iossimulator</TargetOS>
```

Comment:
Good catch, it will simplify the conditions, thanks!

Reply:
Not sure if [Microsoft.NETCore.Native.Unix.targets](https://github.com/dotnet/runtime/pull/82086/files#diff-a8c821001409a0c4d6589a59e663d4d07e53b0127b345f17640410828c580b37) is supposed to be used standalone, but the current approach seems to overwrite the `_IsApplePlatform` project property in that .targets file. 
We could either add a `... and '$(_IsApplePlatform)' == ''` condition to it, or assume it was set in the parent project file?

Reply:
It is only used in `Microsoft.NETCore.Native.targets` so it should be set there. However, to allow extensibility it might be beneficial add a `... and '$(_IsApplePlatform)' == ''` as well so it doesn't need to be set in a parent project file.